pkgs/development/libraries/gtk/3.x.nix

```
@@ -32,6 +32,8 @@
 , sassc
 , x11Support ? stdenv.isLinux
 , waylandSupport ? stdenv.isLinux
+, trackerSupport ? stdenv.isLinux
+, tracker
 , mesa
 , wayland
 , wayland-protocols
```

```suggestion
, mesa
, wayland
, wayland-protocols
, trackerSupport ? stdenv.isLinux
, tracker
```

Probably nicer to keep the wayland things together.

---

Thanks, it should not actually even go to `propagatedBuildInputs`.